lib/rules/selector-max-class/__tests__/index.js

```diff
@@ -100,48 +102,66 @@ testRule({
 			message: messages.expected('.ab.cd.ef', 2),
 			line: 1,
 			column: 1,
+			endLine: 1,
+			endColumn: 10,
 		},
 		{
 			code: '.ab .cd .ef {}',
 			description: 'compound selector: greater than max classes',
 			message: messages.expected('.ab .cd .ef', 2),
 			line: 1,
 			column: 1,
+			endLine: 1,
+			endColumn: 12,
 		},
 		{
 			code: '.ab, \n.cd.ef.gh {}',
 			description: 'multiple selectors: greater than max classes',
 			message: messages.expected('.cd.ef.gh', 2),
 			line: 2,
 			column: 1,
+			endLine: 2,
+			endColumn: 10,
 		},
 		{
 			code: ':not(.ab.cd.ef) {}',
 			description: ':not(): greater than max classes, inside',
 			message: messages.expected('.ab.cd.ef', 2),
 			line: 1,
 			column: 6,
+			endLine: 1,
+			endColumn: 15,
 		},
 		{
 			code: '.ab.cd.ef :not(.gh) {}',
 			description: ':not(): greater than max classes, outside',
 			message: messages.expected('.ab.cd.ef :not(.gh)', 2),
 			line: 1,
 			column: 1,
+			endLine: 1,
+			endColumn: 20,
 		},
 		{
 			code: '.ab { &:hover > .ef.gh {} }',
 			description: 'nested selectors: greater than max classes',
 			message: messages.expected('.ab:hover > .ef.gh', 2),
 			line: 1,
 			column: 7,
+			endLine: 1,
+			endColumn: 26,
+			// TODO: The following case is actually correct. It's hard to get the original selector before resolved.
```

Comment:
Why don't you move it to the accepted ones but skip it instead ?

Reply:
If specifying `skip: true`, the case itself is skipped, and `message` or `column` also is not tested.